### PR TITLE
2/3

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2023-02-03.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+### Embedding Go in C
+
+[Daylon](https://www.dolthub.com/team#daylon) wrote a [great blog](https://www.dolthub.com/blog/2023-02-01-embedding-go-in-c/) and [sample application](https://github.com/dolthub/go-library-sample) explaining how to embed Golang code in a C application. 
+
+This is especially important to us for the [embedded use case](https://www.dolthub.com/blog/2022-07-25-embedded/) where you use Dolt in your application without a running server. Right now, [that's easy to do in Golang](https://www.dolthub.com/blog/2022-07-25-embedded/) but we want to be able to do it in every language, a la [SQLite](https://sqlite.org/index.html). We think the offline first, online sync capabilities of Dolt make a lot of sense in some SQLite use cases. Let us know if that is you by replying to this email.
+
+### Binlog Replication Preview
+
+I did not get a single reply to [last weeks email pitch on binlog replication](https://mailchi.mp/dolthub.com/tims-weekly-dolthub-update-c2e18x7n0u-9291563). This one is cool and important. I will not be denied. Let me tell you again.
```
Comment (bpf120) [inline emails/2023-02-03.md:21]:
```suggestion
I did not get a single reply to [last week's email pitch on binlog replication](https://mailchi.mp/dolthub.com/tims-weekly-dolthub-update-c2e18x7n0u-9291563). This one is cool and important. I will not be denied. Let me tell you again.
```

File: emails/2023-02-03.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+[Jason](https://www.dolthub.com/team#jason) has a preview version of binlog replication from MariaDb or MySQL to Dolt. Turn it on and every transaction commit on your production MySQL instance becomes a Dolt commit on the Dolt replica. You get an audit log of every cell, time travel, and branch for development on a replica. No risk to production. Dolt features at almost zero cost. Imagine how much easier life would have been [at Atlassian](https://www.dolthub.com/blog/2022-04-14-atlassian-outage-prevention/) if they had one of these.
```
Comment (bpf120) [inline emails/2023-02-03.md:23]:
```suggestion
[Jason](https://www.dolthub.com/team#jason) has a preview version of binlog replication from MariaDB or MySQL to Dolt. Turn it on and every transaction commit on your production MySQL instance becomes a Dolt commit on the Dolt replica. You get an audit log of every cell, time travel, and branch for development on a replica. No risk to production. Dolt features at almost zero cost. Imagine how much easier life would have been [at Atlassian](https://www.dolthub.com/blog/2022-04-14-atlassian-outage-prevention/) if they had one of these.
```

Comment (bpf120) [inline emails/2023-02-03.md:23]: You could say something about using Hosted Dolt after 'Dolt features at almost zero cost'.

File: emails/2023-02-03.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+We need people to help test it with us. You just configure a Dolt replica in the same way you would a MySQL replica. Reply to this email if you want to help. Pretty please. Next week I'm coming back with an image if I don't get a reply.
```
Comment (bpf120) [inline emails/2023-02-03.md:25]: maybe list the steps here to show simple it is and recommend a hosted size based on their DB size. could be for next week too with your image.

File: emails/2023-02-03.md (hunk 2 above)
Comment (timsehn) [reply emails/2023-02-03.md:23]: Can't use it in Hosted Dolt yet though

**Pull request merged**